images/gptscript/config/main.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. gptscript)."
+  default     = ["gptscript"]
```

Comment:
Should we toss in BusyBox and a few other utilities here? Gptscript can run programs locally to download and manipulate files, so we should probably include enough tools for it to be able to be useful.

Reply:
We can certainly add `busybox`. Maybe `python` as well?

Reply:
What about `apk-tools` so it can install arbitrary packages and take over the world?

Reply:
Good idea. Then everyone can just swap in what they need

Reply:
Darren confirmed it just needs ca-certs and glibc, but that a few other packages like curl and busybox would be helpful. Agree with adding in apk-tools too!